package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "3.1.4",
+  "version": "3.1.6",
```

Comment:
There is no need to skip a patch version.

Reply:
... Even if you wait for Alex's PR to be merged first (he sets the version to 3.1.5), you will have to rebase before this PR is ready to merge.

Reply:
I'll change it to 3.1.5. I was thinking Alex's PR will be merged before mine.